### PR TITLE
add models :CinemaHall, MovieSession and implemented dao and service …

### PR DESCRIPTION
**Pull request by RomanDubovskyi**
…layers for them. Tested method in Main class.

### CODE REVIEW

File: src/main/java/com/dev/cinema/Main.java [hunk 1]
```diff
@@ -12,6 +19,24 @@ public static void main(String[] args) {
         movie.setTitle("Fast and Furious");
         MovieService movieService = (MovieService) injector.getInstance(MovieService.class);
         movieService.add(movie);
-        movieService.getAll().forEach(System.out::println);
+        //movieService.getAll().forEach(System.out::println);
```
Comment (Tarabarnik) [inline src/main/java/com/dev/cinema/Main.java]: коментарі з кодом заважають читати реальний код

File: src/main/java/com/dev/cinema/service/impl/CinemaHallServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    public CinemaHall add(CinemaHall cinemaHall) {
+        return cinemaHallDao.add(cinemaHall);
+    }
+
+    @Override
+    public List<CinemaHall> getAll() {
+        try {
+            return cinemaHallDao.getAll();
+        } catch (DataProcessingException e) {
+            LOGGER.error("Can't get all Halls" + e);
```
Comment (Tarabarnik) [inline src/main/java/com/dev/cinema/service/impl/CinemaHallServiceImpl.java]:
```suggestion
            LOGGER.error("Can't get all halls", e);
```

File: src/main/java/com/dev/cinema/Main.java (hunk 1 above)
Comment (RomanDubovskyi) [reply src/main/java/com/dev/cinema/Main.java]: sry, will consider this in future

File: src/main/java/com/dev/cinema/service/impl/CinemaHallServiceImpl.java (hunk 2 above)
Comment (RomanDubovskyi) [reply src/main/java/com/dev/cinema/service/impl/CinemaHallServiceImpl.java]: done

**Review by Tarabarnik — APPROVED** [review]

**Pull request merged**